### PR TITLE
Add selected_columns_and_types method

### PR DESCRIPTION
**Pull request by sneaxiy**
The method `selected_columns_and_types` would be used by the `verifier` module in the Python side after refactory.

### CODE REVIEW

File: python/runtime/db.py [hunk 1]
```diff
@@ -192,29 +192,97 @@ def read_feature(raw_val, feature_spec, feature_name):
...
+    """
+    if n < 0:
+        return select
+
+    def replace_limit_num(matched_limit):
+        num = int(matched_limit.group(1))
+        return "LIMIT {}".format(min(num, n))
+
+    if LIMIT_PATTERN.search(select) is None:
+        return select + " LIMIT {}".format(n)
```
Comment (typhoonzero) [inline python/runtime/db.py:217]: If the statement got `;` at the end?

File: python/runtime/db.py [hunk 2]
```diff
@@ -192,29 +192,97 @@ def read_feature(raw_val, feature_spec, feature_name):
...
+MYSQL_DATA_TYPE_DICT = {
+    1: "TINYINT",
+    3: "INT",
+    4: "FLOAT",
+    5: "DOUBLE",
+    8: "BIGINT",
+    246: "DECIMAL",
+    252: "TEXT",
+    253: "VARCHAR",
+    254: "CHAR",
```
Comment (lhw362950217) [inline python/runtime/db.py:231]: How about use MySQLdb.constants.FIELD_TYPE constants instead of the numbers?

**Review by typhoonzero — APPROVED** [review]
Cool!

File: python/runtime/db.py (hunk 2 above)
Comment (sneaxiy) [reply python/runtime/db.py:231]: Thanks for your suggestions! Would fix it in the next PR.

File: python/runtime/db.py (hunk 1 above)
Comment (sneaxiy) [reply python/runtime/db.py:217]: Thanks for your suggestions! Would fix it in the next PR.

**Pull request merged**